eng/versioning/external_dependencies.txt

```diff
@@ -227,3 +229,7 @@ media_com.microsoft.azure:adal4j;1.2.0
 
 # sdk\storage\azure-storage-blob-cryptography\pom.xml
 storage_com.microsoft.azure:azure-storage;8.4.0
+
+# sdk\spring\azure-spring-boot\pom.xml
+spring_io.micrometer:micrometer-core;1.3.0
```

Comment:
/CC @kushagraThapar 
The only library that uses io.micrometer:micrometer-core and io.micrometer:micrometer-registry-azure-monitor dependencies is Cosmos. Could cosmos also be updated to use the 1.3.0 versions? I would prefer that, whenever possible, that we update the common version of an external dependency rather than declare library specific ones.

Reply:
Yes, I don't see a reason why cosmos can't update the micrometer dependency to 1.3.0 version. As you mentioned @JimSuplizio  this is not a blocker for this PR, and can be done later, but we should definitely have one common version.